lib/index.js

```diff
@@ -45,13 +45,16 @@ function generate_google_oauth2_url () {
 
 exports.register = function googleauth (server, options, next) {
   OPTIONS = options;
+  OPTIONS.config = OPTIONS.config || {};
+  OPTIONS.config.auth = false;
```

Comment:
@Y-LyN-10 @nelsonic This might be good, shall we open an issue with a `discuss` label in the backlog?